app/src/main/java/inaka/com/testapp/MainActivity.java

```diff
@@ -0,0 +1,15 @@
+package inaka.com.testapp;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+import inaka.com.testapp.R;
+
+public class MainActivity extends AppCompatActivity {
```

Comment:
According to **Lewis**:
> 
| Id                  | Severity | Category    |
|---------------------|----------|-------------|
| RootPackage | Error | Correctness |
 **Message**:  Expecting MainActivity not to be in root package inaka.com.testapp.
  **Explanation**: Every .java file should be inside a custom package inside the root package..